### PR TITLE
updated direct travis file to bring in line with PP and tidied up tec…

### PR DESCRIPTION
**Pull request by CharlotteSpinks**
…h debt

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,49 +7,39 @@ cache:
...
+services:
+ - mysql
+ - postgresql
 
 env:
  global:
-  - IGNORE_PATHS=sdk/,fonts/,pix/
-  - IGNORE_NAMES=jquery*,moment.js,colorbox.css,tooltipster.css,*.min.css,*.min.js,tii-icon-webfont.css,styles.css,styles.css.map,gulpfile.js
+  - IGNORE_PATHS=sdk/,pix/,vendor/
+  - IGNORE_NAMES=jquery*,styles.css,autoload_static.php
```
Comment (jmcgettrick) [inline .travis.yml]: Switch these back to what they were. We need to ignore all of those

File: .travis.yml [hunk 2]
```diff
@@ -7,49 +7,39 @@ cache:
...
+
+services:
+ - mysql
+ - postgresql
 
 env:
  global:
-  - IGNORE_PATHS=sdk/,fonts/,pix/
-  - IGNORE_NAMES=jquery*,moment.js,colorbox.css,tooltipster.css,*.min.css,*.min.js,tii-icon-webfont.css,styles.css,styles.css.map,gulpfile.js
+  - IGNORE_PATHS=sdk/,pix/,vendor/
```
Comment (jmcgettrick) [inline .travis.yml]: As per comment below. Remove vendor

File: .travis.yml (hunk 1 above)
Comment (CharlotteSpinks) [reply .travis.yml]: Done :D

File: .travis.yml (hunk 2 above)
Comment (CharlotteSpinks) [reply .travis.yml]: Done :D

**Pull request merged**